packages/clerk-js/src/ui/elements/Menu.tsx

```diff
@@ -194,6 +194,9 @@ export const MenuItem = (props: MenuItemProps) => {
           justifyContent: 'start',
           borderRadius: theme.radii.$sm,
           padding: `${theme.space.$1} ${theme.space.$3}`,
+          ':focus': {
+            boxShadow: 'none',
```

Comment:
We can do focusRing={false} instead of this.

Reply:
Fixed!